fmf_scheduler.py

```diff
@@ -0,0 +1,115 @@
+import os
```

Comment:
What does this script is suppose to be doing?

Reply:
it does this scheduler, it dynamically generates nosetest testclasses and cases from metadata

Reply:
I still don't understand how should I use this file:
```
$ ./fmf_scheduler.py
zsh: permission denied: ./fmf_scheduler.py

$ python3 ./fmf_scheduler.py
/home/tt/.local/lib/python3.6/site-packages/requests/__init__.py:80: RequestsDependencyWarning: urllib3 (1.23) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Traceback (most recent call last):
  File "./fmf_scheduler.py", line 16, in <module>
    TARGET = Target(os.environ.get("TARGET"), LOG_LEVEL)
  File "/home/tt/g/user-cont/colin/colin/core/target.py", line 78, in __init__
    self.instance = Target._get_target_instance(target, logging_level=logging_level)
  File "/home/tt/g/user-cont/colin/colin/core/target.py", line 100, in _get_target_instance
    if os.path.isfile(target):
  File "/usr/lib64/python3.6/genericpath.py", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

$ python3 ./fmf_scheduler.py registry.access.redhat.com/rhscl/mariadb-101-rhel7:latest
/home/tt/.local/lib/python3.6/site-packages/requests/__init__.py:80: RequestsDependencyWarning: urllib3 (1.23) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Traceback (most recent call last):
  File "./fmf_scheduler.py", line 16, in <module>
    TARGET = Target(os.environ.get("TARGET"), LOG_LEVEL)
  File "/home/tt/g/user-cont/colin/colin/core/target.py", line 78, in __init__
    self.instance = Target._get_target_instance(target, logging_level=logging_level)
  File "/home/tt/g/user-cont/colin/colin/core/target.py", line 100, in _get_target_instance
    if os.path.isfile(target):
  File "/usr/lib64/python3.6/genericpath.py", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```

Reply:
@TomasTomecek I've wrote it to PR description. usage could be via nosetests like:
```
TARGET=tests/data/Dockerfile nosetests -d -v fmf_scheduler.py
```
and I've fixed it, and `fmf_scheduler.py` can be used as scheduler too and fixed there issue when envvar `TARGET` is not given